apps/docs/scripts/generateComponentData.mts

```diff
@@ -54,28 +54,57 @@ function getFormattedData(data: ComponentDoc[]): ComponentDocWithGroups[] {
 
     const groupsConfig: GroupsConfig = {
         events: "Events",
-        a11y: 'Aria',
+        a11y: ["Aria", "Focusable"],
```

Comment:
I think I prefer the group header that react-aria uses: "Accessibility". Not sure about everyone else though.